### PR TITLE
Next/60x/20230130/v6

### PR DESCRIPTION
**Pull request by victorjulien**
Staging:
- #8490 
- #8492
- #8498 
- #8499

suricata-verify-pr: 1101

### CODE REVIEW

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 12153

File: src/detect-engine-event.c [hunk 1]
```diff
@@ -173,8 +173,13 @@ static DetectEngineEventData *DetectEngineEventParse (const char *rawstr)
     }
 
     if (OutdatedEvent(rawstr)) {
-        SCLogWarning(
-                SC_WARN_DEPRECATED, "decode-event keyword no longer supports event \"%s\"", rawstr);
+        if (SigMatchStrictEnabled(DETECT_DECODE_EVENT)) {
+            SCLogError(SC_WARN_DEPRECATED, "decode-event keyword no longer supports event \"%s\"",
+                    rawstr);
+            goto error;
```
Comment (inashivb) [inline src/detect-engine-event.c:179]: Do we want existing rules to start failing in strict mode?

File: src/detect-engine-event.c [hunk 2]
```diff
@@ -173,8 +173,13 @@ static DetectEngineEventData *DetectEngineEventParse (const char *rawstr)
...
+        } else {
+            SCLogNotice("decode-event keyword no longer supports event \"%s\"", rawstr);
```
Comment (inashivb) [inline src/detect-engine-event.c:181]: Also, is the Notice intentional?

File: src/detect-engine-event.c (hunk 1 above)
Comment (victorjulien) [reply src/detect-engine-event.c:179]: yes, it only happens with `--strict-rule-keywords=decode-event`, not with regular `-T`

File: src/detect-engine-event.c (hunk 2 above)
Comment (victorjulien) [reply src/detect-engine-event.c:181]: Yeah, wanted to reduce the severity as warning is known to be treated similar to error by some. In 7 it is still a warning.

**Pull request merged**